similarity_processor/similarity_io.py

```diff
@@ -1,15 +1,18 @@
 """ This file process the IO for the Text similarity index processor """
 import math
 import os
+import datetime
 import pandas as pd
-from similarity_processor.similarity_core import get_cosine
-from similarity_processor.similarity_core import text_to_vector
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer # pylint: disable=E0401
+from sklearn.metrics.pairwise import cosine_similarity # pylint: disable=E0401
```

Comment:
Please help justify why this pylint warning is disabled?

Reply:
Removed in this instance. but it is present in verify_path.py and sanity_test.py. The lint disable is activated here since we are not packaging these folders and hence there are false positive lint errors so kept the suppression at local.